graph/src/blockchain/mod.rs

```diff
@@ -199,12 +199,10 @@ pub trait TriggerFilter<C: Blockchain>: Default + Clone + Send + Sync {
 pub trait DataSource<C: Blockchain>:
     'static + Sized + Send + Sync + Clone + TryFrom<DataSourceTemplateInfo<C>, Error = anyhow::Error>
 {
-    // ETHDEP: `Mapping` is Ethereum-specific.
-    fn mapping(&self) -> &Mapping;
-
     fn address(&self) -> Option<&[u8]>;
     fn start_block(&self) -> BlockNumber;
 
+    // ETHDEP: These arguments are ethereum-specific
```

Comment:
Is this comment just tagging ethereum dependencies for later refactors?

Reply:
Yhea it got refatored in the following commit.